package/MDAnalysis/analysis/encore/utils.py

```diff
@@ -0,0 +1,390 @@
+# utils.py --- Utility functions for encore package
```

Comment:
Pretty much everything in here is a good candidate for being made generically available under `MDAnalysis.lib`, namely the `log` (logging) and `mdamath` (matrices etc) modules. 

It would make ENCORE leaner, provide others with useful functions, and will make sure that these functions are all tested individually in the context of lib (once the tests are written...).